SAGE Journals.js

```diff
@@ -174,6 +174,9 @@ function scrape(doc, url) {
 				if (reviewType && reviewType.match(/other/i) && item.issue === '5') {
 					item.tags.push('Book Review');
 					item.notes.push({note: "Booklist:" + item.date.match(/\d{4}$/)});
+					if (item.abstractNote && item.abstractNote.match(/,(?!\s\w)/g)) {
+						delete item.abstractNote;	
+					}
```

Comment:
Setting `item.abstractNote` to the empty string should be sufficient and might prevent sid effects.

Reply:
Do you mean like this? `item.abstractNote = '';` But i don't see the difference, cause `delete item.abstractNote;` is inside the if clause.

Reply:
Yes. Being in the if-clause does not affect the scope here because `item` is passed to the callback function for the `itemDone` handler. It is not a "local" object which is why I personally would be careful with removing attributes.

Reply:
Thanks for the explanation. I changed it.

Reply:
Please push.